README.md

```diff
@@ -1,6 +1,6 @@
 # station
 
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/superfaceai/station/CI)
+[![CI / CD](https://github.com/superfaceai/station/actions/workflows/ci_cd.yml/badge.svg)](https://github.com/superfaceai/station/actions/workflows/ci_cd.yml)
 ![TypeScript](https://img.shields.io/badge/%3C%2F%3E-Typescript-blue)
 
 <img src="https://github.com/superfaceai/station/blob/main/docs/LogoGreen.png" alt="superface logo" width="150" height="150">
```

Comment:
The image should point to _raw_ representation for it to be renderable. You may alter the path if you still wish to move it from `.github` directory.

```suggestion
<img src="https://github.com/superfaceai/station/raw/main/.github/docs/LogoGreen.png" alt="superface logo" width="150" height="150">
```

Reply:
Works in https://raw.githubusercontent.com/superfaceai/one-sdk-js/main/README.md https://raw.githubusercontent.com/superfaceai/cli/main/README.md

Reply:
Aah, okay, GitHub does magic to these by changing it to raw automatically. Nevermind then :) 
Just note a logo already exists at `.github/docs/LogoGreen.png` so you might want to use that or remove it in favour of the added one.

Reply:
Ah, thanks didn't know. 👍